<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroUsuario</class>
 <widget class="QWidget" name="RegistroUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lClave">
     <property name="text">
      <string>Clave</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lApellido">
     <property name="text">
      <string>Apellido</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lUsuario">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lNombre">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lMail">
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pbRegistrarse">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="leMail"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="leApellido"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="leNombre"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="leUsuario"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leClave">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbMostrarClave">
     <property name="text">
      <string>Mostrar Clave</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leUsuario</tabstop>
  <tabstop>leClave</tabstop>
  <tabstop>leNombre</tabstop>
  <tabstop>leApellido</tabstop>
  <tabstop>leMail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
